staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -766,6 +766,11 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdownBasedOnPodPriority
 	// +optional
 	ShutdownGracePeriodByPodPriority []ShutdownGracePeriodByPodPriority `json:"shutdownGracePeriodByPodPriority,omitempty"`
+	// CrashLoopBackOff contains config to modify node-level parameters for
+	// container restart behavior
+	// +featureGate=KubeletCrashLoopBackOffMax
+	// +optional
+	CrashLoopBackOff *CrashLoopBackOffConfig `json:"CrashLoopBackOff,omitempty"`
```

Comment:
json name should be "crashLoopBackOff", so when you regenerate openapi, the violation exception will not be needed.

Reply:
my b fixed in 3fb3006! along with other embarrassing stale names